slidez-fe-core/src/common_components/button/GeneralButton.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { MouseEventHandler, ReactNode } from 'react'
+import { Button } from '@material-ui/core'
+import { withStyles } from '@material-ui/styles'
+
+const styles: any = () => ({
+    root: {
+        backgroundColor: '#59d0a5',
+        '&:hover': {
+            backgroundColor: '#73b69e',
+        },
+        color: 'white',
+        fontFamily: "'Rubik', serif",
+        fontSize: '16px',
+        paddingTop: '10px',
+        paddingBottom: '10px',
+        border: '2px transparent solid',
+        borderRadius: '30px',
+        flex: 1,
+        boxSizing: 'border-box',
+        textTransform: 'none',
+        fontWeight: '400',
+    },
+})
```

Comment:
Maybe styles should be moved into a separate file for styles?

Reply:
In reality it can be done so

Reply:
But yeah, move it to scss file, as they are not changed and we use styles files everywhere